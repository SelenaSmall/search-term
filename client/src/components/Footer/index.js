import React from 'react';

export default () => (
  <section id="bottom-region" className="md:fixed">
    <menu className="twitter-menu">
      <ul>
        <li className="twitter-menu-item twitter-menu-item-left status">
          <a href="http://twitter.com/intent/user?screen_name=selenasmall88">
            <img className="twitter-menu-item-avatar" src="/avatar-selena.jpg" alt="selena-small" />
            <span>@selenasmall88</span>
          </a>
        </li>
        <li className="twitter-menu-item status">
          <a href="http://twitter.com/intent/user?screen_name=saramic">
            <img className="twitter-menu-item-avatar" src="/avatar-michael.jpg" alt="michael-milewski" />
            <span>@saramic</span>
          </a>
        </li>
      </ul>
    </menu>
  </section>
);
