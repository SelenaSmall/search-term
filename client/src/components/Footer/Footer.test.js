import React from 'react';
import Footer from './';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

it('basic rendering', () => {
  const output = shallow(<Footer />)
  expect(shallowToJson(output)).toMatchSnapshot()
})
