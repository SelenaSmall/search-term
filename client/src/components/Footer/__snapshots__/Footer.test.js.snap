// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic rendering 1`] = `
<section
  className="md:fixed"
  id="bottom-region"
>
  <menu
    className="twitter-menu"
  >
    <ul>
      <li
        className="twitter-menu-item twitter-menu-item-left status"
      >
        <a
          href="http://twitter.com/intent/user?screen_name=selenasmall88"
        >
          <img
            alt="selena-small"
            className="twitter-menu-item-avatar"
            src="/avatar-selena.jpg"
          />
          <span>
            @selenasmall88
          </span>
        </a>
      </li>
      <li
        className="twitter-menu-item status"
      >
        <a
          href="http://twitter.com/intent/user?screen_name=saramic"
        >
          <img
            alt="michael-milewski"
            className="twitter-menu-item-avatar"
            src="/avatar-michael.jpg"
          />
          <span>
            @saramic
          </span>
        </a>
      </li>
    </ul>
  </menu>
</section>
`;
